### PR TITLE
Portfolio setup

### PR DESCRIPTION
**Pull request by anbehindY**
I added an index.html and styles.css that makes up the main page of the mobile version

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @anbehindY 👋, 

While you made a great effort in this project, I can't go ahead and review your code.

## Invalid Code Review Request

- Kindly use the recommended Microverse's README.md template https://github.com/microverseinc/readme-template. Please provide relevant information for all sections of the README.md file. The FAQ section is the only optional section. 👍 
- Kindly make your pull request summary descriptive. Your summary should describe the features you implemented in this branch 👍  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <title>Portfolio Setup</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <div id="name"><a href="#">John Doe</a></div>
```
Comment (leonardodiasb) [inline index.html:16]: - [ ] You used the correct anchor tag for the logo, but the color of the logo does not match with the template. The `a` tag is a little bit tricky when using parent styles, so I recommend you use the `color: inherit` attribute or add the color directly to the logo `a` tag.
 - Your logo:
 - 
image

 - Template:
 - 
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                    <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.</p>
+                    <div class="socials">
+                        <a class="social" href=""><img src="icons/github.svg" /></a>
+                        <a class="social" href=""><img src="icons/linkedin.svg" /></a>
+                        <a class="social" href=""><img src="icons/angelist.svg" /></a>
+                        <a class="social" href=""><img src="icons/twitter.svg" /></a>
+                        <a class="social" href=""><img src="icons/medium.svg" /></a>
+                    </div>
+            </article>
```
Comment (leonardodiasb) [inline index.html:34]: - [ ] This is the first section of your page and a `section` tag would fit better than an `article` tag. Kindly change that for better HTML semantics.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <header>
+            <div id="name"><a href="#">John Doe</a></div>
+            <nav class="toolbar">
+                <a id="hamburger" href="#"><img src="icons/menu.svg"></a>   
+            </nav>
+        </header>
+        <main>
+            <article class="headline-section">
+                    <h2>Hey There.<br>I’m Alex</h2>
+                    <h4>I’am a Software Developer</h4>
```
Comment (leonardodiasb) [inline index.html:24]: - [ ] The main title("Hey There. I’m Alex") should be an `h1` tag. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. Kindly change that.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @anbehindY 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️
 - Professional README file. ✔️ 
 - Correct use of GitHub flow. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by anbehindY** [comment]
I added index.html and styles.css files that make up the first page of the mobile version of the portfolio.
README.md is also updated.
All the linters are checked and green.

**Pull request merged**